docs/_automation/waste-reduction-logic-app/README.md

```diff
@@ -0,0 +1,71 @@
+
+# Automated detection
+
+The **Waste Reduction Logic App** is an automated detection mechanism provided by [Azure Logic App](https://docs.microsoft.com/en-us/azure/logic-apps/logic-apps-overview). The logic app will run on a configurable schedule to monitor selected subscriptions for a set of idle resources and send notifications once it finds any of those resources to alert admins to investigate and take action.
+
+Use the following steps to deploy the Logic App
+
+1. Create a new **Custom Deployment**
```

Comment:
Can we use CreateUiDefinition like the other templates? That can happen in a separate PR, but this feels overly manual.